app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` là gì em

Reply:
`&.` là toán tử Safe Navigation, giúp code trông đơn giản hơn ạ

Reply:
Hoạt động như nào em

Reply:
Ở đây hoạt động là để kiểm tra xem user có được authenticate là true hay ko ạ,  Nếu không có thì trả về nil

Reply:
Sai rồi nha kiểm tra xem user có tồn tại không và user.authenticate(params[:session][:password]) có true ko nha